### ISSUE TITLE
Packages changed in PR #10053 were not version bumped.

### ISSUE THREAD

**Issue by nathan-muir** [meteor/meteor#10357]
Hi @benjamn ,

I was just tracking down what happened regarding #10320 / https://github.com/meteor/meteor/commit/152879c0933ce5e25e0b0a5d157d9ad264225a67

It appears that when #10053 ( 79ae184 & 677026e ) were merged & tagged in [release/METEOR@1.8](https://github.com/meteor/meteor/releases/tag/release%2FMETEOR%401.8), the versions for ddp-server / id-map / mongo-id  were not incremented.

This resulted in these changes being present in the meteor tool, but not in the released packages. (And caused issues in 1.8.1-beta when some of the packages _were_ version bumped)

---

I'm a bit confused by the release mechanism, as packages seem to be compatible across major releases, even though they're kept in a mono repo.

Is there any tooling that I could contribute to ensure package versions are bumped? (eg, checkout & diff against last release?)

**Comment by nathan-muir** [comment]
Also #10286 / https://github.com/meteor/meteor/commit/8f86f19cd0de6cbe82395a8e402e6d9406670b59 was tagged in https://github.com/meteor/meteor/releases/tag/release%2FMETEOR%401.8.0.1 but was also not version bumped/released.

**Comment by benjamn** [comment]
You're right, the process of bumping package versions is unfortunately manual right now. The [`meteor publish-release`](https://github.com/meteor/meteor/blob/0fcc7ddd46d0ef8a278376ba64538210486ab646/tools/cli/commands-packages.js#L686) command has some logic to compare the last published version of a package to the current version (see [this `somethingChanged` logic](https://github.com/meteor/meteor/blob/0fcc7ddd46d0ef8a278376ba64538210486ab646/tools/cli/commands-packages.js#L978)).

However, to be perfectly honest, the hash-based comparison is pretty noisy because any package we haven't republished in a recent Meteor release will tend to have a different hash, even if nothing has changed, just because the build process has changed slightly during that time. Because of that noise, in practice we use the `--skip-tree-hashing` flag, which effectively disables the check.

An alternative approach might be to make the consistency check more forgiving, perhaps by checking whether there have been any recent commits in a given package directory since the last commit that bumped the package version?

**Comment by nathan-muir** [comment]
It shouldn't be too bad if you can rely on the previous release to bump versions correctly.

A basic one liner will get you the packages with changes since the previous tag.
```sh
git diff --name-only release/METEOR@1.8...HEAD -- packages/ | awk -F"/" '{print $2}' | uniq
```

**Comment by namenotrequired** [comment]
> Also #10286 / 8f86f19 was tagged in release/METEOR@1.8.0.1 but was also not version bumped/released.

Bump! 😄 

https://github.com/meteor/meteor/pull/10286 was never launched, but I have been running it in production (first on 1.11 and then on 1.11.1) for a month with no issue.

**Comment by filipenevola** [comment]
This will be available on Meteor 2.0
